### ISSUE TITLE
Identify FUSE mountpoint as remote 

### ISSUE THREAD

**Issue by touilleMan** [Scille/parsec-cloud#7957]
see https://github.com/libfuse/libfuse/issues/531

tl;dr: On Linux, gnulib uses a special trick with the name of the filesystem to [determine if it's a remote one or not](https://git.savannah.gnu.org/gitweb/?p=gnulib.git;a=blob;f=lib/mountlist.c;h=ca1be63f21ec44f5bee8b1e5400ba391005c5934;hb=HEAD#l229):
```c
#ifndef ME_REMOTE
/* A file system is "remote" if its Fs_name contains a ':'
    or if (it is of type (smbfs or cifs) and its Fs_name starts with '//')
    or if it is of type (afs or auristorfs)
    or Fs_name is equal to "-hosts" (used by autofs to mount remote fs).  */
# define ME_REMOTE(Fs_name, Fs_type)            \
    (strchr (Fs_name, ':') != NULL              \
    || ((Fs_name)[0] == '/'                    \
        && (Fs_name)[1] == '/'                 \
        && (strcmp (Fs_type, "smbfs") == 0     \
            || strcmp (Fs_type, "smb3") == 0   \
            || strcmp (Fs_type, "cifs") == 0)) \
    || strcmp (Fs_type, "afs") == 0            \
    || strcmp (Fs_type, "auristorfs") == 0     \
    || strcmp ("-hosts", Fs_name) == 0)
#endif
```

Complying to this would help avoid file manager to try to scrap the whole mountpoint :hand_with_index_finger_and_thumb_crossed: 

In theory the change should be done here and is quiet trivial (changing it to `parsec://` should be good enough)
https://github.com/Scille/parsec-cloud/blob/5cb813add634f3846c661d3570a9b59ada325407/libparsec/crates/platform_mountpoint/src/unix/mount.rs#L54
